Unity/opentok/holo/holo/Capturer.mm

```diff
@@ -0,0 +1,373 @@
+#import "Capturer.h"
+
+#include <memory>
+
+#import <sdk/objc/base/RTCVideoFrameBuffer.h>
+#import <sdk/objc/components/video_frame_buffer/RTCCVPixelBuffer.h>
+#import <sdk/objc/vonage/capturer/VonageRTCCameraVideoCapturer.h>
+
+#import <AugmentedCompression.h>
+
+#define COMPRESSED_SIZE 320 * 1024
+
+typedef NS_ENUM(int32_t, OTCapturerErrorCode) {
+
+    OTCapturerSuccess = 0,
+
+    /** Publisher couldn't access to the camera */
+    OTCapturerError = 1650,
+
+    /** Publisher's capturer is not capturing frames */
+    OTCapturerNoFramesCaptured = 1660,
+
+    /** Publisher's capturer authorization failed */
+    OTCapturerAuthorizationDenied = 1670,
+};
+
+@interface DepthDataCompressor : NSObject<RTC_OBJC_TYPE(RTCVideoDepthDataDelegate)>
+
+@end
+
+@implementation DepthDataCompressor
+
+- (BOOL)compressInputArray:(const std::unique_ptr<uint8_t[]> &)inputArray inputSize:(uint32_t)inputSize outputArray:(std::unique_ptr<uint8_t[]> &)outputArray outputSize:(uint32_t &)outputSize {
```

Comment:
with the change in webrtc this can be removed.

Reply:
![Screenshot 2024-01-22 at 12 48 25](https://github.com/Vonage/vonage-media-transformers-samples/assets/66061508/2960d227-6560-432b-b566-6fca4529e18e)

It resulted in a new warning, I hope it is okay.